app/models/container_node.rb

```diff
@@ -67,4 +67,12 @@ def event_where_clause(assoc = :ems_events)
   def perf_rollup_parents(interval_name = nil)
     [ext_management_system] unless interval_name == 'realtime'
   end
+
+  def ipaddress
+    labels.find_by_name("kubernetes.io/hostname").try(:value)
+  end
+
+  def cockpit_url
+    "http://#{ipaddress}:9090"
```

Comment:
please, build the URL the same way @chessbyte suggested in my PR: https://github.com/ManageIQ/manageiq/pull/10083/files#diff-f157bfd6489d2b7c579915792eb3372fR17